run-dev.sh

```diff
@@ -18,9 +18,9 @@ trap ctrl_c INT
 
 python manage.py runserver --insecure &
 runserver_pid=$!
-python manage.py sass $SOURCE_SCSS $TARGET_CSS --watch -t compressed &
-sasswatch_pid=$!
+#python manage.py sass $SOURCE_SCSS $TARGET_CSS --watch -t compressed &
+#sasswatch_pid=$!
 
 wait ${runserver_pid} > /dev/null 2>&1
-wait ${sasswatch_pid} > /dev/null 2>&1
+#wait ${sasswatch_pid} > /dev/null 2>&1
```

Comment:
imho this whole script should just go away as we can rely on `npm start:both` to achieve the same.

Reply:
Already removed but not pushed